scripts/apps/authoring/attachments/AttachmentsEditorDirective.ts

```diff
@@ -0,0 +1,5 @@
+
+export default function AttachmentsEditorDirective() {
```

Comment:
Is this supposed to be an empty directive for a React component? If so, you can use `reactToAngular1`

Reply:
it renders a template